### PR TITLE
Hardcoded string fix

### LINKED ISSUE #22: Remove Hardcoded Strings and Texts
**Describe the chore**
There are about 84 Hard coded Strings and 30 Hard coded text in the whole project it would be really nice if these resources are extracted.

**Would you like to work on the issue?** (Yes/No)
No

**Screenshots**

Screenshot from 2019-06-10 20-35-34
Screenshot from 2019-06-10 20-36-01

### PR DESCRIPTION
**Pull request by adarshsiva**
Fixed #22

Changes: Removed hardcoded strings

### CODE REVIEW

**Comment by adarshsiva** [comment]
@yashk2000  done

**Comment by yashk2000** [comment]
Great, please squash the commits.

File: TempleApp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -6,6 +6,110 @@
...
+    <string name="REG">REG</string>
+    <string name="WAIT">Hey Wait Please...</string>
+    <string name="JSON">I am getting your JSON</string>
+    <string name="json_obj">Json obj</string>
+    <string name="result">result</string>
+    <string name="selected_donate">Selected To Donate Money</string>
+    <string name="selected_pooja">Selected To Register New Pooja</string>
+    <string name="REG">REG</string>
+    <string name="DON">DON</string>
+    <string name="WAIT">Hey Wait Please...</string>
```
Comment (Chromicle) [inline TempleApp/src/main/res/values/strings.xml]: Great work, but you given multiple times for same string like`DON` you have given more than 4 time please avoid that

Comment (adarshsiva) [reply TempleApp/src/main/res/values/strings.xml]: done

Comment (Chromicle) [reply TempleApp/src/main/res/values/strings.xml]: Hey, still there are so many strings like for string `Update` you wrote two times `update` and `update_`  please make sure that for each string there will be only one resource

**Comment by adarshsiva** [comment]
removed repeated strings from strings.xml

File: TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java [hunk 2]
```diff
@@ -26,7 +26,7 @@
  */
 public class InsertData extends AppCompatActivity {
 
-    String id,name,poojaTyp,overall,money,paidCheck = "NOT PAID";
+    String id,name,poojaTyp,overall,money,paidCheck = getString(R.string.NOT_PAID);
```
Comment (Chromicle) [inline TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java]: here you're accessing resources too early You can only use your activity as a Context with Resources in onCreate() or later in the activity lifecycle
so it producing null pointer exception please change it

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java [hunk 3]
```diff
@@ -163,33 +163,33 @@ protected Void doInBackground(Void... params) {
 
                                 JSONObject innerObject = array.getJSONObject(jIndex);
 
-                                String id = innerObject.getString("ID");
-                                String name = innerObject.getString("NAME");
+                                String id = innerObject.getString(getString(R.string.ID));
```
Comment (Chromicle) [inline TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java]: Here, it is `ID` and `NAME` but you changed them to `ID:` and `NAME:` so fetching of data does not occur

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java [hunk 4]
```diff
@@ -163,33 +163,33 @@ protected Void doInBackground(Void... params) {
...
-                                    if (id.substring(0, 3).equals("REG") && str[2].equals("PAID")) {
+                                    if (id.substring(0, 3).equals(getString(R.string.REG)) && str[2].equals(getString(R.string.PAID))) {
                                         model.setName(name);
                                         model.setCountry(id.substring(3, id.length()));
                                         list.add(model);
 
                                     }
                                 } else if (flag == 4) {
-                                    if (id.substring(0, 3).equals("DON") && str[2].equals("NOT PAID")) {
+                                    if (id.substring(0, 3).equals(getString(R.string.DON)) && str[2].equals(getString(R.string.NOT_PAID))) {
```
Comment (Chromicle) [inline TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java]: here it is `REG` I unexpedtly written DON please change that

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadSingleData.java [hunk 5]
```diff
@@ -102,27 +102,27 @@ protected void onPreExecute() {
...
             try {
                 /*
                   Check Whether Its NULL???
                  */
                 if (jsonObject != null) {
 
-                    JSONObject user = jsonObject.getJSONObject("user");
-                    name = user.getString("name");
+                    JSONObject user = jsonObject.getJSONObject(getString(R.string.user));
+                    name = user.getString(getString(R.string.name));
```
Comment (Chromicle) [inline TempleApp/src/main/java/example/TempleApp/Activities/ReadSingleData.java]: here it is `NAME` not `NAME:` please change that

File: TempleApp/src/main/res/layout/insert_data.xml [hunk 6]
```diff
@@ -103,7 +103,7 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
                 android:layout_marginTop="45dp"
-                android:text="Insert" />
+                android:text="@string/insert" />
```
Comment (Chromicle) [inline TempleApp/src/main/res/layout/insert_data.xml]: Here it is `INSERT` not `Inserting your values`    please make sure that it does not disturb any behavior of app and please check whether all the functionalities are working or not

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java (hunk 3 above)
Comment (adarshsiva) [reply TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java]: Done

File: TempleApp/src/main/res/layout/insert_data.xml (hunk 6 above)
Comment (adarshsiva) [reply TempleApp/src/main/res/layout/insert_data.xml]: I have corrected that

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadSingleData.java (hunk 5 above)
Comment (adarshsiva) [reply TempleApp/src/main/java/example/TempleApp/Activities/ReadSingleData.java]: Made corrections

File: TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java (hunk 4 above)
Comment (adarshsiva) [reply TempleApp/src/main/java/example/TempleApp/Activities/ReadAllData.java]: Changed it to REG

File: TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java (hunk 2 above)
Comment (yashk2000) [reply TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java]: @adarshsiva please revert this change.

Comment (adarshsiva) [reply TempleApp/src/main/java/example/TempleApp/Activities/InsertData.java]: done

**Comment by vchrombie** [comment]
Awesome work, @adarshsiva.
Thanks for completing all the requested changes with patience. :slightly_smiling_face: 

I am merging this as it is approved by both. :tada: 
Keep up the good work.
Looking forward to your contributions.

**Pull request merged**